sdk/identity/azure-identity-broker/src/main/java/com/azure/identity/broker/package-info.java

```diff
@@ -3,6 +3,20 @@
 
 /**
  * <p>The Azure Identity Brokered Authentication library offers support for using various support brokers from the
- * MSAL4j library.</p>
+ * MSAL4j library. Current support:</p>
```

Comment:
Will be helpful to explain the concept of brokered authentication here and under which situations it is helpful for the users.